docs/articles/AbstractionLayer.md

```diff
@@ -66,8 +66,7 @@ The [Workplan](xref:Workplans) package of the AbstrationLayer contains several h
 
 ## Tutorials
 
-You can find some tutorials here:
+To start using the AbstractionLayer for your own stuff you can jump into these tutorials:
 
-* [How to create a Resource](xref:HowToCreateAResource)
-* [How to build a Driver](xref:HowToBuildADriver)
-* [How to use Constraints](xref:HowToUseConstraints)
+* [How to create a Resource](/docs/articles/Tutorials/HowToCreateResource.md)
+* [How to build a Driver](/docs/articles/Tutorials/HowToBuildADriver.md)
```

Comment:
No pathes to the md files should be used. We use docfx for generating the documentation. To instead of using the direct path to the files. `xref:UUID` should be used. The UUID is part of the target markdown (header attribute)

Reply:
I know we had docfx and plan on adding it again, but right now we have a lot of broken links which just make browsing navigation harder. And even if we bring it back, UUID does not work locally and does not work for anyone who just browses the repo. Since docfx [also supports relative file links](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html) I suggest dumping UUID and using plain markdown.

Reply:
Removing the uids has the following disadvantages: 

- Sometimes there are links between pathes that are very long and hard to understand
- API references cannot be linked by path and **must have** a uid
- Refactorings in documentation are getting harder because you must know which file is linking to the moved file
- The plan was to generate a complete documentation for moryx and not for each project itself. So you must know pathes from other repos. 
- Our documentation gets bigger an bigger and browing it by files is confusing anyway

We should definitly work on hosting the documentation and link from this repo. Have a look at projects like https://github.com/dotnet/docs - it is a very well structured documentation. It can be browsed in repository but links also not work.

Reply:
I suggest that @dev-lutz made his changes compliant to the whole documentation and that this topic will be discussed within issue https://github.com/PHOENIXCONTACT/MORYX-Platform/issues/21 to finish this PR